readme.md

```diff
@@ -247,6 +247,35 @@ entities:
       <extra></extra>
 ```
 
+### Offsets
+
+```yaml
+type: custom:plotly-graph
+offset: 6h
+entities:
+  - entity: sensor.weather_24h_forecast
+    offset: -1d
+  - entity: sensor.actual_temperature
+```
+
+### Extend_to_present
```

Comment:
@FrnchFrgg 
Here's the toggle for `extend_to_present` with the defaults you suggested